src/compose.js

```diff
@@ -14,9 +14,9 @@ export default function compose(...funcs) {
     return arg => arg
   }
 
-  if (funcs.length === 1) {
-    return funcs[0]
+  if (funcs.length===1) {
+  return funcs[0]
   }
 
-  return funcs.reduce((a, b) => (...args) => a(b(...args)))
+  return  funcs.reduce((a, b)=> (...args) => a(b(...args)))
```

Comment:
format: style mismatch:
```suggestion
  return  funcs.reduce((a, b) => (...args) => a(b(...args)))
```

`ea11337e` ␣ should be inserted at column 30.